src/app/common/http-client.ts

```
@@ -80,11 +80,12 @@ export class HttpClient {
 
     public request(url: string, options?: RequestOptionsArgs, warn?: boolean): Promise<Response> {
         return this.requestOnConnection(url, options, warn).pipe(
-            take(1),
-            catchError(e => {
-                if (this.runtime.HandleConnectError(e)) {
-                    return throwError(e);
+            catchError((e, _) => {
+                var unhandled = this.runtime.HandleConnectError(e);
+                if (unhandled) {
+                    return throwError(unhandled);
                 }
+                return of(null);
```

please add some comment why we ignore exception error here.